data/urls.json

```diff
@@ -2,5 +2,7 @@
     "root": "/",
     "index": "/tao/Main/index",
     "login": "/tao/Main/login",
-    "logout": "tao/Main/logout"
+    "logout": "tao/Main/logout",
+    "nosplashParam": "nosplash=true",
```

Comment:
These are not really urls, just query parameters. I think if we would like to use something like this, it should be separated into another file.

Reply:
Moved in to `urlParams.json`